### PR TITLE
Don't show profile settings in embedded mode

### LINKED ISSUE #1179: Element Call embedded mode: Clean up Settings Menu
In embedded mode some of the Element Call settings options do not make sense:
- Profil

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-call/issues/1179

### CODE REVIEW

File: src/settings/SettingsModal.tsx [hunk 1]
```diff
@@ -59,6 +60,8 @@ interface Props {
 export const SettingsModal = (props: Props) => {
   const { t } = useTranslation();
 
+  const { isEmbedded } = useUrlParams();
```
Comment (toger5) [inline src/settings/SettingsModal.tsx:63]: I remember to also have seen some `!widget` logit for detecting embedded mode? Is this the proper way. Is this also consistend with all other code?

**Review by toger5 — APPROVED** [review]
Looks good (I added one question)

File: src/settings/SettingsModal.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/settings/SettingsModal.tsx:63]: Hmm, so as far as I can tell both are used and I am not sure if their usage differs

Comment (SimonBrandner) [reply src/settings/SettingsModal.tsx:63]: For posterity: we decided to go with `isEmbedded` as it makes more sense for UI

**Pull request merged**